test/smoke-tests/coronal-spectra.jl

```diff
@@ -0,0 +1,26 @@
+using Test
+using Gradus
+
+m = KerrMetric(1.0, 0.998)
+d = GeometricThinDisc(0.0, 100.0, π / 2)
+
+model = LampPostModel(h = 10.0)
+
+# test usual case
+em_prof1 = Gradus.emissivity_profile(
+    m,
+    d,
+    model,
+)
+
+# try explicitly setting a spectrum
+spectrum = Gradus.PowerLawSpectrum(3.0)
+
+# now with spectrum as an argument
+em_prof2 = Gradus.emissivity_profile(
+    m,
+    d,
+    model,
+    spectrum,
+)
```

Comment:
Missing some explicit numerical tests. The simplest you can do is make sure it's working right, then do a low-resolution calculation and ensure that it is always able to reproduce that (see some of the other tests for examples).